packages/web-client/src/helpers/share/role.ts

```diff
@@ -409,6 +409,12 @@ export abstract class LinkShareRoles {
       return bitmask === (bitmask | r.bitmask(false))
     })
   }
+
+  static getByName(name: string, isFolder: boolean): ShareRole {
+    return LinkShareRoles.list(isFolder, true, true, true).find(
```

Comment:
Well, you need to use the actual capabilities here 😄

Reply:
oh

Reply:
You do it similar to like it's done here: https://github.com/owncloud/web/blob/master/packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue#L559 (though you need to call `getByName()` instead of `list()`). In `getByName()` you then need to pass the params to `list()`.

Reply:
done